sections/05_treat.md

```diff
@@ -162,45 +162,193 @@ feature selection and construction.*
+graphs for undirected molecular graphs by creating an ensemble of directed
+graphs in which one atom is selected as the root node [@tag:Lusci2013_rnn].  A
+single feature vector is obtained by summing over all feature vectors for all
+directed graphs in the ensemble.  Graph convolutions on undirected molecular
+graphs have eliminated the need to enumerate artificial directed graphs,
+learning feature vectors for atoms that are a function of the properties of
+neighboring atoms and local regions on the molecular graph
+[@tag:Kearnes2016_graph_conv @tag:AltaeTran2016_one_shot]. `TODO: add more
+details on how graph convolutions work?`
+
+Advances in chemical representation learning have not been limited to
+graph-based neural networks.  The simplified molecular-input line-entry system
+(SMILES) is a standard way to transform a chemical into string-based
+representation.  A SMILES-to-SMILES autoencoder learns a continuous latent
+feature space for chemicals [@tag:Gomezb2016_automatic].  In this learned
+continuous space is it possible to train some types of supervised learning
+algorithms or interpolate between continuous representations of chemicals in a
+manner that is not possible with discrete (e.g. bit vector or string) features.
+A drawback is that not all SMILES strings produced by the autoencoder's decoder
+correspond to valid chemical structures. `TODO: could mention GANs here`
+Altae-Tran et al developed a one-shot learning network
+[@tag:AltaeTran2016_one_shot] to address the reality that most practical
+chemical screening studies are unable to provide the thousands or millions of
+training compounds that are needed to train larger multitask networks.  Using
+graph convolutions to featurize chemicals, the network learns an embedding from
+compounds into a continuous feature space such that compounds with similar
+activities in a set of training tasks have similar embeddings.  The approach is
+evaluated in an extremely challenging setting where the embedding is learned
+from a subset of prediction tasks (e.g. activity assays for individual proteins)
+and only one to ten labeled examples are provided as training data on a new
+task.  On Tox21 targets, even when trained with _one_ task-specific active
+compound and _one_ inactive compound, the model is able generalize reasonably
+well because it has learned an informative embedding function from the related
+tasks. Random forests, which cannot take advantage of the related training
+tasks, trained in the same setting achieve near random performance.  Despite the
+success on Tox21, performance on MUV datasets, which contains assays designed to
+be challenging for chemical informatics algorithms, is considerably worse.  The
+authors also demonstrate the limitations of transfer learning as embeddings
+learned from the Tox21 assays have little utility for a drug adverse reaction
+dataset.
+
+In the long term, these novel, learned chemical feature representations may
```

Comment:
Good point.  The `long term` here indicates that novel featurizations have not yet proved themselves in practical applications, but have the potential to do so with continued research.  I dropped `long term` here and the rest of the paragraph conveys the idea.